docs/topics/http/file-uploads.txt

```diff
@@ -122,6 +122,54 @@ field in the model::
             form = UploadFileForm()
         return render(request, 'upload.html', {'form': form})
 
+Uploading multiple files
+------------------------
+
+If you want to upload multiple files at the same time, you need to set the
+``multiple`` boolean attribute of the input element in your
+:class:`~django.forms.ModelForm` first:
+
+.. snippet::
+    :filename: forms.py
+
+    from django import forms
+    from .models import ModelWithFileField
+
+    class FileFieldModelForm(forms.ModelForm):
+
+        file_field = forms.FileField(widget=forms.ClearableFileInput(attrs={'multiple': True}))
```

Comment:
Is this field meant to correspond a field on the model? I would think the `FileField` model field doesn't really work with multiple files, does it? Maybe an example that uses `Form` and a note about this would be appropriate.